proto/iotics/api/meta.proto

```diff
@@ -36,6 +36,28 @@ service MetaAPI {
   // 1. http://data.iotics.com/graph#custom-public (aka custom public graph) - All metadata written to this graph will be
   //    visible during SPARQL queries both with local & global scope (and thus, the Iotics network).
   rpc SparqlUpdate(SparqlUpdateRequest) returns (SparqlUpdateResponse) {}
+
+  // ExplorerQuery - Deprecated - use SparqlQuery instead.
+  rpc ExplorerQuery(ExplorerRequest) returns (stream SparqlQueryResponse) {}
+}
+
+// ExplorerRequest - Deprecated. Use SparqlQueryRequest instead.
+message ExplorerRequest {
+
+  // Explorer request payload.
+  message Payload {
```

Comment:
Actually may want resultContentType in here too - similar to sparqlQueryRequest.
Easier for both parties imo... @StephanieBracaloni

Reply:
Good idea, but I know there are restrictions with the type of request and the resultContentType (If the request is a describe all the results types are not available)

We can verify with MFT if they are always using the same result type or not (for explorer)

Reply:
We are using rdf n triples because we are doing a describe, but I think it would be good to have resultContentType just in case

Reply:
Nice one, will add this in.